graph/docu/docu_04graph_02plotutils.cc

```diff
@@ -67,7 +67,9 @@ for (size_t e = 0; e < 2; ++e) {
     .OutputFormat("svg");
 #endif
 if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, (std::string("golden/love.") + extensions[e]).c_str());
+#ifndef BRICKS_APPLE // TODO(dkorolev): Figure out how to run this test on Apple.
 if (!e) ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString(std::string("golden/love.") + extensions[e]));
```

Comment:
If I understand correctly, it does not make any sense to use this code at all without this tool installed. Much better would be change #ifdef to instructions how to install it.

> On Mar 3, 2015, at 15:37, John Babak <notifications@github.com> wrote:
> 
> In graph/docu/docu_04graph_02plotutils.cc <https://github.com/KnowSheet/Bricks/pull/108#discussion_r25683994>:
> 
> > @@ -67,7 +67,9 @@ for (size_t e = 0; e < 2; ++e) {
> >      .OutputFormat("svg");
> >  #endif
> >  if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, (std::string("golden/love.") + extensions[e]).c_str());
> > +#ifndef BRICKS_APPLE // TODO(dkorolev): Figure out how to run this test on Apple.
> >  if (!e) ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString(std::string("golden/love.") + extensions[e]));
> Easier to not depend on any tool that "can be easily installed via ...".
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KnowSheet/Bricks/pull/108/files#r25683994>.
>